src/main/java/core/basesyntax/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+
+public class ReturnOperation implements OperationHandler {
+    public void handle(FruitTransaction transaction) {
+        Storage.add(transaction.getFruit(), transaction.getQuantity());
```

Comment:
What happens if quantity is negative?) Let's check it in all other handlers and throw an exception with informative message in this case. You may need to add a default or static method to the OperationHandler to avoid code duplication.